airbyte-commons-protocol/src/test/java/io/airbyte/commons/protocol/AirbyteMessageSerDeProviderTest.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.commons.protocol;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import io.airbyte.commons.protocol.serde.AirbyteMessageDeserializer;
+import io.airbyte.commons.protocol.serde.AirbyteMessageSerializer;
+import io.airbyte.commons.version.AirbyteVersion;
+import java.util.Optional;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class AirbyteMessageSerDeProviderTest {
+
+  AirbyteMessageSerDeProvider serDeProvider;
+  AirbyteMessageDeserializer<String> deserV0;
+  AirbyteMessageDeserializer<String> deserV1;
+
+  AirbyteMessageSerializer<String> serV0;
+  AirbyteMessageSerializer<String> serV1;
+
+  @BeforeEach
+  void beforeEach() {
+    serDeProvider = new AirbyteMessageSerDeProvider();
+
+    deserV0 = buildDeserializer(new AirbyteVersion("0.1.0"));
+    deserV1 = buildDeserializer(new AirbyteVersion("1.1.0"));
+    serDeProvider.registerDeserializer(deserV0);
+    serDeProvider.registerDeserializer(deserV1);
+
+    serV0 = buildSerializer(new AirbyteVersion("0.2.0"));
+    serV1 = buildSerializer(new AirbyteVersion("1.0.0"));
+    serDeProvider.registerSerializer(serV0);
+    serDeProvider.registerSerializer(serV1);
+  }
+
+  @Test
+  void testGetDeserializer() {
+    assertEquals(Optional.of(deserV0), serDeProvider.getDeserializer(new AirbyteVersion("0.1.0")));
```

Comment:
Same nit for assertJ
it should look like:

assertThat(serDeProvider.getDeserializer(new AirbyteVersion("0.1.0"))).hasValue(deserV0)